manifests/setup.pp

```diff
@@ -46,10 +46,6 @@
     default   => $script_path
   }
 
-  File {
-    backup => false,
-  }
```

Comment:
Instead of changing the default, could you just change the backup value for the `File[$script_path]` resource?